test/testnet/localnet/README.md

```diff
@@ -0,0 +1,94 @@
+# Remote Version of Localnet
+
+Remote version of the test network generated by running:
+
+```bash
+> make localnet-start
+```
+
+from the Tendermint repository. There are a few additional minor tweaks, however:
+
+1. The proxy app is preconfigured to `kvstore`
+2. Logging format is JSON
+3. `persistent_peers` is preconfigured
```

Comment:
The point of localnet - and I would argue that the same is true for this remotenet setup - is that the developer can set configuration as they see fit. The `tendermint testnet` command was used to generate the minimum viable default configuration as a template for the developer but it is expected to be changed before a network is started. In that Thai ID issue for example, create_empty_blocks has to be set to true explicitly, if I recall correctly. Persistent peers can't really be set automatically as soon as you create a new testnet.

Instead of manually tweaking the configuration during setup (which gets cumbersome very quickly as well as not really useful for the developer), I think we should stick to the default config for now and figure out an overall method how to improve applying the config to an arbitrary testnet. This requires a bit of discussion and use-cases from the dev team.